packages/driver/cypress/e2e/e2e/origin/patches.cy.ts

```diff
@@ -62,9 +63,7 @@ describe('src/cross-origin/patches', { browser: '!webkit' }, () => {
     describe('from the AUT', () => {
       beforeEach(() => {
         cy.intercept('/test-request').as('testRequest')
-        cy.origin('http://www.foobar.com:3500', () => {
-          cy.stub(Cypress, 'backend').callThrough()
-        })
```

Comment:
The stubs for backend have been moved out of the origin call and are now stubbed on the primary cypress instance because we will now call into the primary cypress instance through post message to call the backend method.

Reply:
Yeah no users should be using it or trying to stub it.